test/config/test_configtypes.py

```diff
@@ -1600,32 +1582,32 @@ def test_validate_len_ok(self):
     def test_validate_len_toolong(self):
         """Test validate with a too long length."""
         t = configtypes.WebKitBytesList(length=3)
-        with self.assertRaises(configexc.ValidationError):
+        with pytest.raises(configexc.ValidationError):
             t.validate('1,2,3,4')
 
     def test_transform_single(self):
         """Test transform with a single value."""
-        self.assertEqual(self.t.transform('1k'), [1024])
+        assert self.t.transform('1k') == [1024]
 
     def test_transform_more(self):
         """Test transform with multiple values."""
-        self.assertEqual(self.t.transform('23,2k,1337'), [23, 2048, 1337])
+        assert self.t.transform('23,2k,1337'), [23, 2048 == 1337]
```

Comment:
No please... that's just a regex/replace gone wrong, your detailed review is certainly appreciated. I will see to fix those tomorrow. :smile: